janusgraph-hadoop-parent/janusgraph-hadoop-core/src/main/java/org/janusgraph/hadoop/formats/util/JanusGraphVertexDeserializer.java

```diff
@@ -219,10 +219,7 @@ public TinkerVertex readHadoopVertex(final StaticBuffer key, Iterable<Entry> ent
 
         /*Since we are filtering out system relation types, we might end up with vertices that have no incident relations.
          This is especially true for schema vertices. Those are filtered out.     */
-        if (!foundVertexState) {
-            log.trace("Vertex {} has unknown lifecycle state", vertexId);
-            return null;
-        } else if (!tv.edges(Direction.BOTH).hasNext() && !tv.properties().hasNext()) {
+        if (!tv.edges(Direction.BOTH).hasNext() && !tv.properties().hasNext()) {
```

Comment:
Why did you remove the log ?
And can you delete the dead code just above ?

Reply:
@davidclement90 This section is unreachable because it's conditioned on `!foundVertexState` and `foundVertexState` is set to `!verifyVertexExistence` ([L93](https://github.com/JanusGraph/janusgraph/blob/84ead7d907b62061e30c5d3210e5e8422fd4648c/janusgraph-hadoop-parent/janusgraph-hadoop-core/src/main/java/org/janusgraph/hadoop/formats/util/JanusGraphVertexDeserializer.java#L93)), which is always true since `verifyVertexExistence` is `false` ([L48](https://github.com/JanusGraph/janusgraph/blob/84ead7d907b62061e30c5d3210e5e8422fd4648c/janusgraph-hadoop-parent/janusgraph-hadoop-core/src/main/java/org/janusgraph/hadoop/formats/util/JanusGraphVertexDeserializer.java#L48)). I never would have found this if it wasn't for Coverity (and it took me awhile to see it even after it was flagged).

I just pushed updates to further remove the useless `foundVertexState`, which hopefully makes this clearer. Also removed the commented out code.